videoretrieval/datasets/msrvtt/constants.py

```diff
@@ -54,12 +54,20 @@
 densenet_path = precomputed_features_base_path + \
 	"aggregated_scene_25fps_256px_stride1_offset0/densenet161-avg.pickle"
 
+audio_path = precomputed_features_base_path + \
+	"aggregated_audio_feats/Audio_MSRVTT_new.pickle"
+
```

Comment:
nit: I think removing vertical whitespaces between these features and making it more compact might would help readabibility.